### ISSUE TITLE
last sentence here cut off?

### ISSUE THREAD

**Issue by kdkorthauer** [mtello22/KorthauerLab#2]
https://github.com/mtello22/KorthauerLab/blob/3df8ec2c497323b50ef302e0ee84beee70f23dbf/VMRassociation/SupervisedLearning/Reports/CommitteeReport/CommitteeReport.Rmd#L274C530-L274C530

**Comment by mtello22** [comment]
Yes, I wanted to plot the shuffled results, but the plots look clearer without them.  
On the same note, would it be best to show the results from a single model per regression method? Maybe change the boxplot to bar plots, one for each donor. You mentioned it would be good to present validation results using a single model, should I change it?

**Comment by kdkorthauer** [comment]
While if you were to actually use the model in practice you'd need to pick one, keep the boxplots as they help evaluate the cv error across models

**Comment by kdkorthauer** [comment]
Good to still at least mention the negative control like you do for other results that aren't presented with 'data not shown'
